.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
     rev: v1.8.0
     hooks:
     - id: mypy
-      exclude: database
+      pass_filenames: false
```

Comment:
The size of the python bindings is very small, so might as well run mypy on all sources defined in `pyproject.toml`